src/components/NavigationFolder.vue

```diff
@@ -59,6 +58,21 @@
 				>
 					{{ t('mail', 'Add subfolder') }}
 				</ActionInput>
+
+				<ActionButton
+					v-if="debug && !account.isUnified && folder.specialRole !== 'flagged'"
+					icon="icon-settings"
+					:title="t('mail', 'Clear cache')"
+					:disabled="clearingCache"
+					@click="clearCache"
+				>
+					{{
+						t(
+							'mail',
+							'Clear all cached data. Useful when you experience problems with synchronization. Use with care.'
```

Comment:
Looks good, but why the "use with care"? Possibly shorter and more explanatory:

> Clear locally cached data, in case there are issues with synchronization.

Reply:
Because it deletes a lot of data that has to be recreated. Users shouldn't do this just for fun. But they will do that anyway, so :+1: on the wording